tests/MultiRecordSaveAndEditTest.php

```diff
@@ -28,6 +28,14 @@ public function testSaveAndEditNewRecords_NoSort()
             "admin/pages/edit/EditForm/field/"."HasManyRelation"."/addinlinerecord/MultiRecordField_HasManyTest/new/field/".key($_FILES)."/upload",
             $_FILES
         );
+        if ($response->getBody() == "I can't handle sub-URLs on class CMSPageEditController.") {
+            // Note Stephen (2018-05-29) The current SilverStripe 3 branch expects the following url format instead.
+            // It includes the page ID in the URL e.g: "admin/pages/edit/EditForm/1/field/"
```

Comment:
This comment doesn't accurately describe the problem.
Can you adjust slightly to say something like:
"In versions SilverStripe 3.6 and lower, there is no page ID passed to an EditForm action. However in 3.7 (`3` branch at time of writing), this has been changed"